bitnami/kibana/9/debian-12/rootfs/opt/bitnami/scripts/libkibana.sh

```diff
@@ -46,11 +46,12 @@ kibana_create_system_user() {
     local -r retries="60"
     local -r sleep_time="5"
     local url
-    url=$(kibana_sanitize_elasticsearch_hosts "${KIBANA_ELASTICSEARCH_URL}" "${KIBANA_ELASTICSEARCH_PORT_NUMBER}")
+    local first_elastic_node="${KIBANA_ELASTICSEARCH_URL%%,*}"
+    url=$(kibana_sanitize_elasticsearch_hosts "${first_elastic_node}" "${KIBANA_ELASTICSEARCH_PORT_NUMBER}")
```

Comment:
As first_elastic_node is not used anywhere else, what about removing the variable?

```suggestion
    # Connecting to the first host to create system user
    url=$(kibana_sanitize_elasticsearch_hosts "${KIBANA_ELASTICSEARCH_URL%%,*}" "${KIBANA_ELASTICSEARCH_PORT_NUMBER}")
```

Reply:
Yes I totally agree, i just created this variable for readability but adding a comment is also fine :)
Using the first node only to create the kibana system user is the correct way to do it though.

I know it's surely more works, but the "kibana_sanitize_elasticsearch_hosts()" should maybe be renamed "kibana_sanitize_elasticsearch_host()" without a "s" at the end because it does not seems to work as intended when passing multiple hosts as argument, or maybe I did not understood its wole purpose.

thanks a lot for your update.